### ISSUE TITLE
Bug? Freeze on back up phrase after password

### ISSUE THREAD

**Issue by chriise** [bitpay/copay#5959]
First post , please be kind. Just signed up to mention an issue I've come across.

Go to Settings > Wallet > Back up > Got it > 'Screenshots are not secure' I understand > Enter spending password > OK

This reaches my 12-word back up phrase, but at this point I can not click either Back or 'I've written it down'. I have to close BitPay and restart. 

I only noticed this issue after I setup a Spending Password.

I can replicate every time. Not sure if anyone else has come across this issue.

Using version v3.1.3 on Windows 10.

**Comment by chriise** [comment]
Further info:

I tried to view 12-word back up phrase immediately after setting up spending password for the first time, just to test my password out. 

The first time, it took me to the 12-word page, with the password prompt over the top. I could still see my 12 words at this point without entering a password. I thought this was strange. At this point I couldn't click out. I had to close and  restart Bitpay.

After restarting, I could not reach my 12-words without using the password - Good news! But still have the freeze issue at this point.
